src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content">
+    <div class="month month--month-length--31">
+      <div class="month__day month__day--start-day--sun"></div>
```

Comment:
You should add all modifiers to the calendar block 
![image](https://github.com/mate-academy/layout_calendar/assets/99403198/bb360f84-96a5-4673-bf2b-9fbe1f2655a0)

```suggestion
    <div class="calendar calendar--month-length--31 calendar--start-day--sun">
      <div class="calendar__day"></div>
```